Makefile

```diff
@@ -64,6 +64,30 @@ install:
 		helm upgrade $(HELM_OPTS) $(CHART_INSTANCE) $(CHART_DIR); \
 	fi;
 
+.PHONY: install_giraffe
+install_giraffe: ## Installs the helm chart on the OpenShift cluster
+install_giraffe: check_environment
+install_giraffe:
+install_giraffe: GIT_SHA1=$(shell git rev-parse HEAD)
+install_giraffe: IMAGE_TAG=$(GIT_SHA1)
+install_giraffe: NAMESPACE=$(OBPS_NAMESPACE_PREFIX)-$(ENVIRONMENT)
+install_giraffe: CHART_DIR=./helm/cas-registration
+install_giraffe: CHART_INSTANCE=cas-registration
```

Comment:
Would we want to change the `CHART_INSTANCE` to something else, just to avoid colissions if we ever chose to deploy to the same namespace as Moose? I've been using `cas-bciers` in my testing, not sure if that's best though.